mixpanel/mixpanel.d.ts

```diff
@@ -26,7 +26,7 @@ interface Mixpanel
 
     unregister(propertyName:string):void;
 
-    identify(id:string):void;
+    identify(id?:string):void;
```

Comment:
https://mixpanel.com/help/reference/javascript-full-api-reference#mixpanel.identify
`Arguments unique_id String A string that uniquely identifies a user` is this right?